### DISCUSSION TITLE
offer low-end plan fast, before removing hobby

### DISCUSSION THREAD

**Discussion by syffs** [planetscale/discussion#624] [Feature Requests] [answered]
TBH, this is quite unacceptable to get a 1 month notice on a free tier retirement, and have the first paid plan offered at 39$/month (or more depending on the region) ! 
It feels just like a scam.

Please offer a really low-end plan like 5-9$/month max, just like mongodb atlas.

Otherwise, once we've migrated out, I'm telling you some of us are never ever coming back to your service.

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @syffs,

If you let me know which database before Monday, I can extend you for another month to make the transition easier on you.

We have no plans at the moment to offer another plan. Sorry this is disappointing for you. Please know this was not something we took lightly, we did try to find a way to make it work. 

We wish we could support running databases for every workload. But we're currently not a great fit for low traffic/free applications. It would take significant investment for us to support those use cases. Even our smallest Vitess cluster needs 3 running nodes. A load balancer, a primary and a replica. Being able to offer Vitess as low as $39 was already a significant achievement for us.

I'd recommend looking at self-hosting MySQL. Or using sqlite. Both options should work great for your application.
